spec/integration/dependency_resolution_spec.rb

```diff
@@ -61,6 +61,8 @@
     end
   end
 
+
```

Comment:
I think there are a couple of errant blank lines here.

Reply:
Thanks! fixed